pkg/convert/json_nglocalize.go

```diff
@@ -42,16 +43,34 @@ func FromNgLocalize(data []byte, original *bool) (model.Translation, error) {
 	}
 
 	for k, v := range ng.Translations {
+		val, err := toMF2(v)
+		if err != nil {
+			return model.Translation{}, fmt.Errorf("convert string to MF2: %w", err)
+		}
```

Comment:
I dont think we need function for that, ignore prev comment
```suggestion
msg, err := mf2.NewBuilder().Text(v).Build()
if err != nil {
return ...
}
```